ankdown/ankdown.py

```diff
@@ -283,7 +286,9 @@ def main():
     """Run the thing."""
 
     arguments = docopt(__doc__, version=VERSION)
-
+    css_config = arguments.get('--css')
+    if css_config is not None:
+        Card.MODEL_CSS = css_config
```

Comment:
Wouldn't it be better to pass a filename?
```suggestion
        Card.MODEL_CSS = open(css_config, 'r').read()
```
(you'd also want to change the docopt docs of course)
CSS can get long and difficult to embed into a string on the command line.

Reply:
Ah; I see your [other fork](https://github.com/p1tt1/ankdown-plugin). For that use case it seems like a bit more hassle to get this working if it's a file; I still think I favor that approach to keep the CLI clean. Another possible (probably overengineered) approach would be to have a more general configuration file format that (for now) has only CSS. e.g. `--config=[filename]` would override files in XDG_CONFIG_HOME; maybe `--with-config-value=k:v` would override particular settings and could be used for passing CSS in a string

Reply:
A config file would be useful in any case if more attributes can be changed in the future. When using ankdown-plugin it would be conceivable to just load the plugin config file in ankdown. Then you would not have to create a reference to an ankdown config file in the plugin config file ...